src/core/js/offlineStorage.js

```diff
@@ -27,6 +27,14 @@ define([
             this._handler = handler;
         },
 
+        /**
+         * Flag to indicate if an offlineStorage handler has been defined.
+         * @returns {boolean} true if an offlineStorage handler has been defined, false otherwise
+         */
+        hasHandler: function() {
+            return !_.isUndefined(this._handler);
```

Comment:
`_.isUndefined` is an expensive function call, does it help with readability?
```js
  _.isUndefined = function(obj) {
    return obj === void 0;
  };
```
These are cheaper:
```js
return this._handler !== undefined;
return this._handler !== void 0;
```